src/main/objects/HarmonicFunction.js

```diff
@@ -19,8 +20,8 @@ function HarmonicFunction2(params){
     // system               "open" | "close" | undefined
     // mode                 "major" | "minor"
     // key                  string, f.e. "C#", "g#", "Ab"
-
-    var cm = new ChordComponentManager.ChordComponentManager();
+    this.cm = new ChordComponentManager.ChordComponentManager();
```

Comment:
dopisałbym jakiś komantarz, dlaczego to zostało tak zrobione (że to workaround spowodowany musescore, czy coś), żeby uniknąć możliwości popsucia tego w przyszłości

Reply:
Dopisane